src/Uno.UI/UI/Xaml/Controls/TimePicker/TimeSpanExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Windows.UI.Xaml.Controls
+{
+	internal static class TimeSpanExtensions
+	{
+		internal static TimeSpan RoundToInterval(this TimeSpan time, int interval)
```

Comment:
Add /// summary.

Reply:
The description should specify the unit for the `interval` parameter.

Reply:
summary to the class or to the method.?

i can add but i don't see why for both the class and the method the name is self explanatory. 

also i'm moving the method to Uno.UI.Extensions.TimeSpanExtensions

Reply:
on the method.